app/main.py

```diff
@@ -2,4 +2,67 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
You could write this condition without `is`
```suggestion
        self.coords = coords if coords else [0, 0]
```
Or using `or`
```suggestion
        self.coords = coords or [0, 0]
```